shared/apiGeneral/introduction.md

```diff
@@ -0,0 +1,9 @@
+## Introduction
+
+The API is the core of the resin.io platform. It provides a secure channel for communication between resin.io services and the database. The API's HTTP interface not only informs the dashboard and the CLI, it also gives you the power to directly access resources associated with your account. With the API, you can fetch and update information about your applications, devices, environment variables and more.
+
+This guide is split into two parts. On this page, you will find a basic tutorial to help you construct API calls. The [Resources][resources] page provides more details about the resources that can be queried using the API, including example calls and a list of available fields.
+
+__Warning__: When using the API to make changes, take great care in selecting the appropriate resources, as there are *no checks* to prevent you from accidentally making widespread, irreversible mistakes. Test filters with a `GET` call *before* you use them in a `PATCH` or `DELETE` request.
```

Comment:
Would we also want to suggest that people consider creating a backup? We'd had people ask us to restore from our own backups when their API `PATCH` etc went wrong (was used without a filter), which is a huge hassle :/

Reply:
I'm not sure if the API is something they could backup in any straightforward way. Are you thinking maybe copy the current state of a resource before they make a change to it?